pkg/virt-api/webhooks/defaults.go

```diff
@@ -49,9 +49,20 @@ func SetDefaultVirtualMachineInstance(clusterConfig *virtconfig.ClusterConfig, v
 	v1.SetObjectDefaults_VirtualMachineInstance(vmi)
 	setDefaultHypervFeatureDependencies(&vmi.Spec)
 	setDefaultCPUArch(clusterConfig, &vmi.Spec)
+	setGuestMemoryStatus(vmi)
 	return nil
 }
 
+func setGuestMemoryStatus(vmi *v1.VirtualMachineInstance) {
+	if vmi.Spec.Domain.Memory != nil &&
+		vmi.Spec.Domain.Memory.Guest != nil {
+		vmi.Status.Memory = &v1.MemoryStatus{
+			GuestAtBoot:  vmi.Spec.Domain.Memory.Guest,
```

Comment:
> Do we also set this on VM level? 
No, only VMI.

>What is the expected behavior in case I run a VM, hotplug a memory, and then stop/start it? Should the base be the new (hotpluged) memory or the old?

Since you edit the VM itself the expectation is that the new VM will have the new memory value as the base.

> A good candidate for a test imho.

Indeed, I'll create one.

Reply:
e2e test added.